example/session/src/main/java/org/apache/iotdb/SessionExample.java

```diff
@@ -44,9 +44,15 @@ public static void main(String[] args)
     session.open();
 
     session.setStorageGroup("root.sg1");
-    session.createTimeseries("root.sg1.d1.s1", TSDataType.INT64, TSEncoding.RLE, CompressionType.SNAPPY);
-    session.createTimeseries("root.sg1.d1.s2", TSDataType.INT64, TSEncoding.RLE, CompressionType.SNAPPY);
-    session.createTimeseries("root.sg1.d1.s3", TSDataType.INT64, TSEncoding.RLE, CompressionType.SNAPPY);
+    if (session.checkTimeseriesExists("root.sg1.d1.s1")) {
+      session.createTimeseries("root.sg1.d1.s1", TSDataType.INT64, TSEncoding.RLE, CompressionType.SNAPPY);
+    }
+    if (session.checkTimeseriesExists("root.sg1.d1.s2")) {
+      session.createTimeseries("root.sg1.d1.s2", TSDataType.INT64, TSEncoding.RLE, CompressionType.SNAPPY);
+    }
+    if (session.checkTimeseriesExists("root.sg1.d1.s3")) {
+      session.createTimeseries("root.sg1.d1.s3", TSDataType.INT64, TSEncoding.RLE, CompressionType.SNAPPY);
+    }
```

Comment:
This is not necessary in this PR, right? [1]
https://github.com/apache/incubator-iotdb/pull/824/files

Reply:
Updated.